x/cronos/keeper/ibc.go

```diff
@@ -155,22 +157,19 @@ func (k Keeper) IbcTransferCoins(ctx sdk.Context, from, destination string, coin
 	return nil
 }
 
-func (k Keeper) ibcSendTransfer(ctx sdk.Context, sender sdk.AccAddress, destination string, coin sdk.Coin) error {
-	// We extract the channel id from the coin denom
-	channelDenom := ""
+func (k Keeper) ibcSendTransfer(ctx sdk.Context, sender sdk.AccAddress, destination string, coin sdk.Coin, channelId string) error {
 	if types.IsSourceCoin(coin.Denom) {
-		// If coin is source, we can only transfer to crypto.org chain
-		// we are using the cro denom to extract the channel id
-		// TODO: support arbitrary channel?
-		channelDenom = k.GetParams(ctx).IbcCroDenom
+		if channelId == "" || !channeltypes.IsValidChannelID(channelId) {
```

Comment:
seems regex already include empty [check](https://github.com/cosmos/ibc-go/blob/v5.0.0/modules/core/04-channel/types/keys_test.go#L29)
```suggestion
		if !channeltypes.IsValidChannelID(channelId) {
```

Reply:
fixed